lib/modelState.py

```diff
@@ -40,6 +40,11 @@ def disable_thread(cls):
     @classmethod
     def push_context(cls, context: GPTMessageItem):
         """Add the selected text to the stored context"""
+        if context.get("type") != "text":
```

Comment:
This ensures that images don't get inserted into the system messages since that would cause an error